.github/workflows/pr.yml

```diff
@@ -25,8 +29,7 @@ jobs:
       uses: tj-actions/changed-files@v32
       with:
         files: |
-          contrib/**
-          library/**
+          plugins/**
```

Comment:
We'll build/test everything if Makefile or tests/* change like before for PR builds only.